README.md

```diff
@@ -31,13 +31,23 @@ prepended or appended to all log lines using that context. For example, the
 (OpenTelemetry) TraceID and SpanID, and add them to the log record, while also
 annotating the Span with an error code if the log is at error level.
 
-##### Logger in Context Workflow:
+#### Logger in Context Workflow:
 
 Using `ToCtx` and `Logger` lets us store the logger itself within a context,
 and get it back out again. Wrapper methods `With`/`WithGroup`/`Debug`/`Info`/
 `Warn`/`Error`/`Log`/`LogAttrs` let us work directly with a logger residing
 with the context (or the default logger if no logger is stored in the context).
 
+#### Compatibility with both Slog and Logr
+slog-context is compatible with both standard library [slog](https://pkg.go.dev/log/slog)
+and with [logr](https://github.com/go-logr/logr), which is an alternative
+logging api/interface/frontend.
+
+If only slog is used, only `*slog.Logger`'s will be stored in the context.
+If both slog and logr are used, `*slog.Logger` will be automatically converted
+to a `logr.Logger` as needed, and vice versa. This allows full interoperability
+down the stack and with any libraries that use either slog-context or logr.
```

Comment:
1) let's not rush (or we can do an rc1). I already found at least 1 (minor, because @pohly is awesome) big, and I have proposed funcr support for slog.

2) yes

3) the whole lib or just the PR?  I am known for being opinionated so be careful what you ask for.  :)

Reply:
1. Ok
2. Thanks
3. As much as you have time for. I may not take all advice, but I will consider everything. My priority would be on settling the external api (naming and function signatures) for my whole lib, followed by everything else. The library is still new and still version 0.x.x, so there is room to change things.